README.md

```diff
@@ -8,37 +8,39 @@
 
 <p align="center"><img src="https://raw.githubusercontent.com/hackclub/meta/5243af92814b6daacadd66e1342ad073e023544c/logos/hackedu_letter_opaque.png" alt="Hack Club"/></p>
 <p align="center">
-<b><a href="workshops/README.md">workshops</a></b>
-|
-<b><a href="case_studies/">case studies</a></b>
-|
-<b><a href="meta/labs/">labs</a></b>
+  <a href="https://hackclub.io)">Hack Club</a> is the movement of student-led coding clubs.
 </p>
 
 -------------------------------------------------------------------------------
 
-[Hack Club](https://hackclub.io) is the movement of student-led coding clubs.
-Want to start a Hack Club at your school? Apply over at
-https://hackclub.io/apply and [join our Slack](https://slack.hackclub.io) in the
-meantime. Regardless of whether you're part of Hack Club, we encourage you to
-check out and contribute to this repository.
+## Where to Start:
 
-Important links:
+| Action                                  | Link                                        |
+|-----------------------------------------|---------------------------------------------|
```

Comment:
I think this is a good idea. However, if I delete the header row:

```markdown
| Action                                  | Link                                        |
|-----------------------------------------|---------------------------------------------|
| Chat with us _(we're here almost 24/7)_ | https://slack.hackclub.io                   |
| Start a club                            | https://hackclub.io/apply                   |
| See our workshops                       | http://workshops.hackclub.io                |
```


 it ends up looking like this:

![](https://s3.amazonaws.com/f.cl.ly/items/3n2U460i0M2w11080Q1b/Image%202015-12-04%20at%2012.30.47%20AM.png?v=06c98b05)

Any ideas for how we can get rid of the header row?